src/main/scala/diplomaticobjectmodel/model/ISASpecifications.scala

```diff
@@ -0,0 +1,44 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.diplomaticobjectmodel.model
+
+import freechips.rocketchip.diplomaticobjectmodel.model.BaseExtensions.specifications
```

Comment:
I removed it. It did nothing because the class that it imported is defined in this file now.

Aside: Is there any kind of automated lint checking we can do to detect unused/duplicate imports and even possibly enforcing a sorted import list? I know other languages have linters that can detect that.

Reply:
IntelliJ has a function that checks for them.